packages/webpack/src/runtime/runtime.js

```diff
@@ -229,6 +229,7 @@ const lavamoatRuntimeWrapper = (resourceId, runtimeKit) => {
     policyRequire.nmd = (moduleReference) => {
       if (moduleReference === module) {
         module = __webpack_require__.nmd(module)
+        return module
```

Comment:
For the future would be nice to see separate changes in separate commits, if not PRs.